test/sample_test.yaml

```diff
@@ -73,6 +73,7 @@ spec:
               - dsl_static_type_checking
               - pipeline_transformers
               - secret
+              - sidecar
               - execution_order
               - imagepullsecrets
               #- sidecar
```

Comment:
Can we just uncomment this line? Thanks!

Reply:
For sure.